docs/dev/api_usage_examples/get_latest_import.py

```diff
@@ -0,0 +1,41 @@
+'''
+get_latest_import.py
+'''
+
+from common import AUTH_HEADER, USERNAME, construct_url
+
+import requests as req
+from datetime import datetime
+
+# The AUTH_HEADER token can be any valid token.
+def get_latest_import(username):
+    """Gets the latest import timestamp of a given user.
+
+    Args:
+        username: User to get latest import time of.
+
+    Returns:
+        A Unix timestamp if there's an OK status.
+
+    Raises:
+        An HTTPError if there's a failure.
+        A ValueError if the JSON in the response is invalid.
+        An IndexError if the JSON is not structured as expected.
+    """
+    url = construct_url("latest-import")
+
+    params = {
+        "user_name": username
+    }
+
+    response = req.get(url, params=params, headers=AUTH_HEADER)
```

Comment:
I think most of the code should be in this function.

Something like

```
response = requests.get(
   url,
   params={
       'user_name': username,
   },
   headers={
       'Authorization: 'Token <your token>',
   }
)
```

would be much clearer.

Reply:
Yeah, you're right.